src/pages/accruals/Accruals.spec.jsx

```diff
@@ -0,0 +1,157 @@
+import { fireEvent, screen, waitFor } from '@testing-library/react';
+import {
+  renderWithApplicationContext,
+  defaultApplicationContext,
+} from '../../test/helpers/TestApplicationContext';
+import Accruals from './Accruals';
+import i18n from 'i18next';
+import {
+  getAgreements,
+  getAgreementTargets,
+  getAccruals,
+} from '../../api/services/accrualsService';
+import {
+  agreement,
+  agreementTarget,
+  annualTargetHoursAccrual,
+} from '../../../mocks/mockData';
+import pretty from 'pretty';
+
+window.HTMLElement.prototype.scrollIntoView = jest.fn();
+
+const t = i18n.getFixedT('en', 'common');
+
+jest.mock('../../api/services/accrualsService');
+beforeEach(() => {
+  getAgreements.mockImplementation(() => {
+    return {
+      status: 200,
+      data: agreement,
+    };
+  });
+  getAgreementTargets.mockImplementation(() => {
+    return {
+      status: 200,
+      data: agreementTarget,
+    };
+  });
+  getAccruals.mockImplementation(() => {
+    return {
+      status: 200,
+      data: annualTargetHoursAccrual,
+    };
+  });
+});
+
+describe('Accruals', () => {
+  describe('navigation', () => {
+    it('should contain a link to previous day', () => {
+      const { getByRole } = renderWithApplicationContext(
+        <Accruals />,
+        defaultApplicationContext,
+        '/2022-07-01',
+        '/:date'
+      );
+
+      const previousDayLink = getByRole('link', {
+        name: t('accruals.previousDay'),
+      });
+      expect(previousDayLink.pathname).toBe('/accruals/2022-06-30');
+      fireEvent.click(previousDayLink);
```

Comment:
Why do we click the link after the expect statement?

Reply:
Removed click